src/software/guides/addin-storage.md

```diff
@@ -0,0 +1,384 @@
+
+## Updating Stored AddInData Content
+
+To update stored content, use the SET method on a AddInData object while specifying its GUID and ID. The return value is always null.
+
+As an example, use the [API Runner tool](https://my.geotab.com/sdk/api/apiRunner.html) to perform the following operation:
+
+```javascript
+api.call("Set",
+{
+  "typeName": "AddInData",
+  "entity": {
+    "addInId": "a2C4ABQuLFkepPVf6-4OKAQ",
+    "id": "a6tClQu4iFkuroNPqOQsydg",
+    "groups": [{
+        "id": "GroupCompanyId"
+    }],
+    "details": JSON.stringify({
+            "date": "2016-01-01T00:00:00.000Z",
+            "items": [
+              {"name": "bottles", "price": 12},
+            ],
+            "customer": {"name": "joesmith", "email": "joe@smith.com"}
+    })
+  }
+});
+```
+An efficient way to perform an update in your Add-In&#39;s Javascript code would be as follows:
+
+```javascript
+var myAddInObj = getMyAddInDataFunction();
+
+var myData = myAddInObj.data;
+myData.customer.name = "john";
+myData.customer.email = "johndoe@company.com";
+myAddInObj.data = JSON.stringify(myData);
+
+api.call("Set",
+{
+  "typeName": "AddInData",
+  "entity": myAddInObj
+},
+function (result) {
+  // Set operation should return value of null in result
+  // ..Continue
+});
+```
+
+## Deleting an AddInData Object
+
+An AddInData object is deleted when you specify its ID and the GUID of its related Add-In. The return value is always null.
+
+```javascript
+api.call("Remove",
+{
+  "typeName": "AddInData",
+  "entity": {
+    "addInId": "a2C4ABQuLFkepPVf6-4OKAQ",
+    "id": "a6tClQu4iFkuroNPqOQsydg"
+  }
+});
+```
+
+### **AddInData JSON Restrictions**
+
+The following are the only restrictions on the JSON stored within AddInData objects:
+1. The JSON data for an AddInData object must be 10,000 characters or less.
+2. No property in the JSON data can have the format "geotabXYZ". This naming format is reserved for Geotab use.
+
+### **Additional Notes and Limitations**
+- **Legacy property 'Data'**
+    The AddInData object has been available as a beta feature through several releases and as such, we've made improvements through time. Now that we are officially releasing this feature in 2101, a legacy property we are looking to get rid of is the 'Data' property. This is a string property that is not deserialized as an object when sent over JSON. The newer property, 'Details', deserializes as an object and should be used instead (you do not need to call JSON.parse() on this property). **Partners that have designed their applications to work with the 'Data' property should transition to using 'Details'. In a future release, the 'Data' property will be completely removed.**
+- **Cannot delete properties of objects**
+    
+    All objects properties stored in the JSON can be modified but not deleted.
+    Example:
+        Replacing
```

Comment:
Formatting:
![2020-12-07 10_50_33-sdk_addin-storage md at db97c69209f7d5d95ec833366a75f3c8106d261f · Geotab_sdk](https://user-images.githubusercontent.com/3814700/101372644-1e45f980-387a-11eb-9fd0-5fe02efaccc7.png)

Reply:
Fixed in 764501a to have it properly spaced/new-lined.